_ingest-pipelines/processors/ml-inference.md

```diff
@@ -0,0 +1,158 @@
+| `model_config` | Object    | Optional | Custom configuration options for the ML model.                                                                                                                                                                                                                                                                                                                                                      |
+| `max_prediction_tasks` | Integer       | Optional | The maximum number of concurrent prediction tasks that can run during document ingestion. Default is `10`.                                                                                                                                                                                                                                                                                                     |
+| `description` | String    | Optional | A brief description of the processor.                                                                                                                                                                                                                                                                                                                                                                             |
+| `tag` | String    | Optional | An identifier tag for the processor. Useful for debugging to distinguish between processors of the same type.                                                                                                                                                                                                                                                                                                     |
+| `ignore_missing` | Boolean   | Optional | If `true` and any of the input fields defined in `input_map` or `output_map` are missing, the missing fields are quietly ignored. Otherwise, a missing field causes a failure. Default is `false`.                                                                                                                                                                                                                      |
+| `ignore_failure` | Boolean   | Optional | If `true` and any exceptions occur, then they are quietly ignored and the ingestion continues. Otherwise, an exception can occur and cancel the ingestion. Default is `false`.                                                                                                                                                                                                                                             |
```

Comment:
```suggestion
| `ignore_failure` | Boolean   | Optional | If `true` and any exceptions occur, then they are ignored and ingestion continues. Otherwise, an exception can occur and cancel the ingestion. Default is `false`.                                                                                                                                                                                                                                             |
```